src/jackdaw/serdes.clj

```diff
@@ -110,11 +109,11 @@
             ;;   parameterizing the ctor on the thus altered topic config is
             ;;   fairly janky. There's probably a better way to generally
             ;;   parameterize serdes, but leaving this as is for now.
-            ::key-serde
+            :key-serde
```

Comment:
Do we really want to overwrite the serde key with the resolved implementation? The name spacing prevented that in nascent implementation - `:jackdaw.topic/value-serde` in and `:jackdaw.serdes/value-serde` out